src/neo/SmartContract/Callback.cs

```diff
@@ -0,0 +1,16 @@
+using Neo.VM.Types;
+
+namespace Neo.SmartContract
+{
+    internal abstract class Callback
+    {
+        public readonly StackItem[] Arguments;
```

Comment:
Why does a callback definition have actual parameters? I think this should be a list of formal parameters, so it should be an array of `StackItemType`. Or even we only need the count of parameters.